python/foxglove-sdk-examples/ws-asset-server/main.py

```diff
@@ -0,0 +1,43 @@
+import logging
+import time
+
+import foxglove
+
+
+def asset_handler(uri: str) -> bytes | None:
+    """
+    This will respond to "package://" asset requests from Foxglove by reading files from disk.
+    This example doesn't do any path validation or upward traversal prevention.
+    """
+    asset = None
+    if uri.startswith("package://"):
+        filepath = uri.replace("package://", "", 1)
+        try:
+            with open(filepath, "rb") as file:
+                asset = file.read()
+        except FileNotFoundError:
+            pass
+
+    status = "OK" if asset else "Not Found"
+    logging.debug(f"asset_handler {status}: {uri}")
+    return asset
+
+
+def main() -> None:
+    foxglove.set_log_level(logging.DEBUG)
+
+    server = foxglove.start_server(
+        asset_handler=asset_handler,
+    )
+
+    try:
+        while True:
+            # Send transforms for the model as needed, on a `FrameTransformsChannel`
```

Comment:
stale comment?

Reply:
This was a placeholder instruction to the user; they'll need to publish the frame transforms.

I'm going to follow up with a more complete example, at which point I'll remove this.